cmd/utils/cmd.go

```diff
@@ -312,3 +313,28 @@ func ExportPreimages(db ethdb.Database, fn string) error {
 	log.Info("Exported preimages", "file", fn)
 	return nil
 }
+
+func ensureSufficientMemory(sigc chan os.Signal) {
```

Comment:
It's a bit confusing to have this method named `ensureSufficientMemory`, as it's disk-space, not RAM that's being checked.

Reply:
Adressed in c355ae4c3b6bd709e781d37051814f3586130d9f